src/scss/component/accordion/_defaults.scss

```diff
@@ -0,0 +1,38 @@
+// @graupl/graupl accordion component default values.
+//
+// Generally, these should not be used directly when styling components unless a static value is needed.
+// They are mainly used to provide class selectors, fallbacks for custom properties, or loop values.
+//
+// They should not be used to define direct property values (i.e. font-size, color, etc.).
+// Those should be defined as custom properties in the `_variables.scss` file.
+
+@use "sass:map";
+// TODO: Use default values provided in graupl.
+// Accordion selectors.
+$accordion-selector: ".accordion" !default;
+$accordion-icon-selector: ".accordion-icon" !default;
+$accordion-item-selector: ".accordion-item" !default;
+$accordion-trigger-selector: ".accordion-trigger" !default;
+$accordion-title-selector: ".accordion-title" !default;
+$accordion-panel-selector: ".accordion-panel" !default;
+
+// Accordion properties.
+// $accordion-border-radius: 7px !default;
+// $accordion-border: 2px solid hsl(0deg 0% 52%) !default;
+
+// Accordion trigger properties.
+// $accordion-trigger-background: none !default;
+// $accordion-trigger-text-color: hsl(0deg 0% 13%) !default;
+$accordion-trigger-font-size: 1rem !default;
+$accordion-trigger-font-weight: normal !default;
+$accordion-trigger-text-align: left !default;
+$accordion-trigger-background-active: hsl(0deg 0% 97%) !default;
```

Comment:
I think these don't make sense as defaults. They can just be in the variables file as custom props referencing existing graupl values.

Reply:
I removed a few defaults and kept the label alignment.